background.js

```diff
@@ -287,6 +300,34 @@ $(document).ready(function () {
       }
     });
   }
+  
+  /* method to change the anna status icon on page*/
+  function changeStatus(newStatus) {
+    
+    chrome.storage.local.get(/* String or Array */["statusicon"], function(items){
+      
+      console.log("items.statusicon == undefined: " +(items.statusicon == undefined));
+      console.log("items.statusIcon == \"false\": "+(items.statusIcon == "false") )
+      if((items.statusicon == undefined) ||( items.statusicon == "false") ){
+        newStatus = "noIcon";
+      }
+
+      chrome.tabs.query({ active: true, currentWindow: true }, function (tabs) { 
+
+        var tabid = tabs[0].id;
+        
+          chrome.tabs.executeScript(tabid, {
+            code: 'var status ="'+newStatus+'";'
+          }, function() {
+            chrome.tabs.executeScript(tabid, {
+              file: "js/set_status_icon.js"
+              }, function() {
+              console.log("Set Status icon Script Executed Successfully!");
```

Comment:
This doesnt sound like a good console message.

Why not something like "Status set to active"

Reply:
Done, console message changed to `console.log("Status set to "+newStatus)`